package.json

```diff
@@ -12,12 +12,21 @@
     "now:alias": "now alias",
     "now": "npm run now:deploy && npm run now:alias"
   },
-  "keywords": ["micro", "zeit", "micro service", "unfurl", "open graph", "opengraph", "ogp"],
+  "keywords": [
+    "micro",
+    "zeit",
+    "micro service",
+    "unfurl",
+    "open graph",
+    "opengraph",
+    "ogp"
+  ],
   "author": "Bram Borggreve (github:beeman twitter:beeman_nl)",
   "license": "MIT",
   "dependencies": {
     "micro": "^7.3.3",
     "microrouter": "^2.1.0",
+    "now": "^8.5.2",
```

Comment:
not sure now is needed as a dep there?

Reply:
You're right. Of course not, I will rebase if @beeman is interested in the PR.

Reply:
@linktohack I am interested - if you can change the PR to only include the change in index.js I'll merge it today!

Reply:
It makes a lot of sense to have CORS enabled here.

Reply:
PR updated, by CORS, do you mean [this](https://github.com/possibilities/micro-cors)?